pkg/controllers/report/background/controller.go

```diff
@@ -121,6 +121,7 @@ func NewController(
 }
 
 func (c *controller) Run(ctx context.Context, workers int) {
+	logger.Info("background scan", "interval", c.forceDelay.Abs().String())
```

Comment:
@realshuting are you trying to get this log at level 2 ?

Reply:
Yes, this is printed when Kyvenro starts.

Reply:
@eddycharly - any comments? Do you prefer not setting this at level 2?